### ISSUE TITLE
PR for exposing the 'set' functionality of ProgressBarJS

### ISSUE THREAD

**Issue by robert-j-webb** [null-null-null/ember-progress-bar#7]
Hi, would you be interested in a PR that makes it so you can call the 'set' function on the progress bar before the animate function? Here's the code I wrote for it:

```js
didRender(...args) {
    this._super(...args);

    const { progressBar, progress, setProgress } = getProperties(
      this,
      'progressBar',
      'progress',
      'setProgress'
    );

    if (isPresent(setProgress)) {
      progressBar.set(setProgress);
    }

    progressBar.animate(progress, () => this.sendAction('onAnimationComplete'));
  },
```

I'd write a test for it!

**Comment by briarsweetbriar** [comment]
Sure!

**Comment by briarsweetbriar** [comment]
Thanks for the PR! Just released this under `1.0.0`.
